### DISCUSSION TITLE
Limit/Filter repositories accessible thru a proxied registry

### DISCUSSION THREAD

**Discussion by gauthiersiri** [goharbor/harbor#19841] [Q&A] [answered]
Hi here,
I'd like to know if it's possible to limit the scope of acccessible repositries on a proxied registry.

For example, I have a project proxying dockerhub.
It's working very fine, however i would like to limit what can be accessed.
For example, i'd like to limit what's proxied to the repos https://hub.docker.com/crazymax/*

Is there a way to achieve that?

For example, as a comparison, in Jfrog container registry, when creating a remote registry, you can define a "project ID" that allow you to filter whats accessible.

Thanks!

**Comment by wy65701436** [comment, marked as answer]
Tracking this by https://github.com/goharbor/harbor/issues/19844
